adoptium-frontend-parent/adoptium-frontend-assets/src/main/resources/META-INF/resources/css/style.css

```diff
@@ -2,17 +2,21 @@ body {
     padding: 40px 0;
 }
 
-h1, h2, h3, h4 {
+h1, h2 {
     margin-top: 40px;
 }
 
+h3, h4 {
+    margin-top: 20px;
+}
+
 p {
     margin: 15px 0;
 }
 
-li {
-    margin: 10px 0;
-}
+/*li {*/
```

Comment:
Is this commented out on purpose?  If it can be removed entirely then just delete it :-)

Reply:
Removing it